js/blocked_number_controller.js

```diff
@@ -0,0 +1,85 @@
+/* global , Whisper, storage, ConversationController */
+/* global textsecure: false */
+
+/* eslint-disable more/no-then */
+
+// eslint-disable-next-line func-names
+(function() {
+    'use strict';
+
+    window.Whisper = window.Whisper || {};
+
+    const blockedNumbers = new Whisper.BlockedNumberCollection();
+    window.getBlockedNumbers = () => blockedNumbers;
+
+    window.BlockedNumberController = {
+      getAll() {
+        try {
+          this.load();
+        } catch (e) {
+          console.warn(e);
```

Comment:
```suggestion
          window.log.warn(e);
```
Should try to keep the logging consistent